NotificationSystem/SendSubscriberEmail.cs

```diff
@@ -38,12 +40,15 @@ public static async Task Run(
             log.LogInformation("Creating email message");
             var body = await CreateBody(cloudQueue);
 
+            var timeConstraint = TimeLimiter.GetFromMaxCountByInterval(14, TimeSpan.FromSeconds(1));
```

Comment:
Where does the number 14 come from?  At least define a named constant for it instead of just hard coding the number here.

Reply:
Moved to const